python_modules/libraries/dagster-celery-docker/dagster_celery_docker/executor.py

```diff
@@ -285,15 +285,14 @@ def _execute_step_docker(
 
             res = docker_response.decode("utf-8")
         except docker.errors.ContainerError as err:
+            events = [MetadataEntry("Job image", value=docker_image)]
+            events.extend(
```

Comment:
nit: a conditional `append` is more idiomatic/readable than extend containing a nested conditional